hwdb.d/60-keyboard.hwdb

```diff
@@ -1820,24 +1821,24 @@ evdev:input:b0003v04FEp000D*
  KEYBOARD_LED_CAPSLOCK=0
 
 # Lenovo ThinkPad T430s
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT430s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT430s*
```

Comment:
should probably end in `:*`, not just `*`

Reply:
Hmm, am I completely confused, or did this pattern never work? The key that is used for matching has a bunch of trailing stuff (':rvn…:rn…:rvr…:cvn…:ct…:cvr…:'), and the key doesn't match without the trailing asterisk:
```console
$ systemd-hwdb query 'evdev:atkbd:dmi:bvn_:bvr_:bd_:svnLENOVO:pn_:pvrThinkPadT430s' 
KEYBOARD_LED_CAPSLOCK=0
$ systemd-hwdb query 'evdev:atkbd:dmi:bvn_:bvr_:bd_:svnLENOVO:pn_:pvrThinkPadT430s:'
$ systemd-hwdb query 'evdev:atkbd:dmi:bvn_:bvr_:bd_:svnAcer_:pnPeppy:pvr_'
...
XKB_FIXED_MODEL="chromebook"
```
Those were added in c72102edd50, 16bed3afa1b, and bc33509ea0d. @mfvescovi, @paride, @ltworf: did this work back then and stopped working since?

Reply:
With v239-stable (plain old v239 doesn't want to build because of meson changes), I get the same result as with git master. It seems this never worked.

Reply:
> Those were added in [c72102e](https://github.com/systemd/systemd/commit/c72102edd503cabfb7abeee009e06b823a9717c4), [16bed3a](https://github.com/systemd/systemd/commit/16bed3afa1b916ace5e927392a1baab9dd9ff963), and [bc33509](https://github.com/systemd/systemd/commit/bc33509ea0d688428f809712f67c9536a34b6356). @mfvescovi, @paride, @ltworf: did this work back then and stopped working since?

Actually, never checked. I was awaiting for the feature to be fully supported in GNOME (my DE of choice) but I guess it was never completed. Thus, no change.